### PR TITLE
contributor guide

### PR DESCRIPTION
**Pull request by ascampos**
-need to finish code review guideline section, would love feedback on what to include

### CODE REVIEW

File: docs/dev/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- New features
+
+For significant changes to the repository, it’s important to settle on a design before starting on the implementation. Reaching out to us early will help minimize the amount of possible wasted effort and will ensure that major improvements are given enough attention.
+
+1. **Open an issue.** Open an issue about your bug in this repo.
+2. **Message us on Slack.** Reach out to us to discuss your proposed changes.
+3. **Agree on implementation plan.** Write a plan for how this feature or bug fix should be implemented. Should this be one pull request or multiple incremental improvements? Who is going to do each part?
+4. **Submit a work-in-progress PR** It's important to get feedback as early as possible to ensure that any big improvements end up being merged. Submit a pull request and label it `wip` to start getting feedback.
+5. **Review.** At least one Solo team member should sign off on the change before it’s merged. Look at the “code review” section below to learn about what we're looking for.
+6. **Merge and release!**
```
Comment (Krief) [inline docs/dev/CONTRIBUTING.md]: Are they doing merges or are we?

File: docs/dev/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+- Big bug fixes
+- New features
+
+For significant changes to the repository, it’s important to settle on a design before starting on the implementation. Reaching out to us early will help minimize the amount of possible wasted effort and will ensure that major improvements are given enough attention.
+
+1. **Open an issue.** Open an issue about your bug in this repo.
+2. **Message us on Slack.** Reach out to us to discuss your proposed changes.
+3. **Agree on implementation plan.** Write a plan for how this feature or bug fix should be implemented. Should this be one pull request or multiple incremental improvements? Who is going to do each part?
+4. **Submit a work-in-progress PR** It's important to get feedback as early as possible to ensure that any big improvements end up being merged. Submit a pull request and label it `wip` to start getting feedback.
+5. **Review.** At least one Solo team member should sign off on the change before it’s merged. Look at the “code review” section below to learn about what we're looking for.
```
Comment (EItanya) [inline docs/dev/CONTRIBUTING.md]: nit: one team member has to sign off for the build to be kicked off, which blocks the merge anyway. So should probably say that one team member must review.

Or just write see code review guidelines below as they outline that prcess

File: docs/dev/CONTRIBUTING.md (hunk 1 above)
Comment (rickducott) [reply docs/dev/CONTRIBUTING.md]: Probably the person that approves the PR should merge (b/c we don't want someone contributing additional stuff after the PR gets approved).

File: docs/dev/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+6. **Merge and release!**
+
+### Code review guidelines
+
+It’s important that every piece of code in Gloo is reviewed by at least one Solo team member familiar with that codebase.
+
+1. **CI check** A Solo team member needs to kick off the CI process.
```
Comment (rickducott) [inline docs/dev/CONTRIBUTING.md]: To do this, they will leave a comment '/test'

**Comment by rickducott** [comment]
CI found a spelling error: 

```
Already have image (with digest): gcr.io/cloud-builders/wget
cat: /workspace/ci/../ci/spelling_whitelist_words.txt: No such file or directory
cat: /workspace/ci/../ci/spelling_skip_files.txt: No such file or directory
docs/dev/CONTRIBUTING.md:45:17: "memeber" is a misspelling of "member"
docs/dev/CONTRIBUTING.md:45:17: "memeber" is a misspelling of "member"
docs/dev/CONTRIBUTING.md:45:17: "memeber" is a misspelling of "member"
```

File: docs/dev/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+- New features
+
+For significant changes to the repository, it’s important to settle on a design before starting on the implementation. Reaching out to us early will help minimize the amount of possible wasted effort and will ensure that major improvements are given enough attention.
+
+1. **Open an issue.** Open an issue about your bug in this repo.
+2. **Message us on Slack.** Reach out to us to discuss your proposed changes.
+3. **Agree on implementation plan.** Write a plan for how this feature or bug fix should be implemented. Should this be one pull request or multiple incremental improvements? Who is going to do each part?
+4. **Submit a work-in-progress PR** It's important to get feedback as early as possible to ensure that any big improvements end up being merged. Submit a pull request and label it `wip` to start getting feedback.
+5. **Review.** At least one Solo team member should sign off on the change before it’s merged. Look at the “code review” section below to learn about what we're looking for.
+6. **A Solo team memeber will merge and release!**
```
Comment (rickducott) [inline docs/dev/CONTRIBUTING.md]: member

Comment (ascampos) [reply docs/dev/CONTRIBUTING.md]: lol memeber

**Pull request merged**